Update User Guide - Slider and APPLY DIMENSION on  Filter Widget and  Custom Tick Labels in Chart Axis (#12427)
geosolutions-it/MapStore2: docs/user-guide/widgets.md

<video class="ms-docimage" style="max-width:800px;" controls><source src="../img/widgets/filter-ex.mp4"/></video>

#### Applying Time and Elevation dimensions

> For layers that include a time parameter (`datetime`) or elevation (`number`), you can interact with these dimensions and visualize them on the map using the *Filter Selector* widget.

Reviewer: ```suggestion
The dynamic filter tool can also drive the `time` and `elevation` dimensions of WMS layers, synchronize with the map timeline, or control the _Highlight Selected Time_ option of a chart (when not already driven by the timeline). If the value attribute is of type `datetime` or `number` and the selection mode is set to *Single*, the **Apply Dimension** option becomes available in the **Interactions** tab, given that there is at least one valid target available.

A valid target is either a WMS layer exposing a `time` or `elevation` dimension, the _timeline_ plugin, if present in the current context, or a chart with the _Highlight Selected Time_ option enabled.
```

To enable the **Apply Dimension** option in a filter, follow these steps:

> * In the [Data](widgets.md#data) tab, set the *Values From* option to **Attributes** and add the `datetime` or `number` attribute to the *Value Attribute* field.

Reviewer: ```suggestion
* In the [Data](widgets.md#data) tab, set the *Values From* option to **Attributes** and select a `datetime` or `number` attribute in the *Value Attribute* field.
```

<img src="../img/widgets/time-attribute.jpg" class="ms-docimage"  style="max-width:450px;"/>

* In the [Interactions](widgets.md#interactions) tab, click the <img src="../img/button/++++.jpg" class="ms-docbutton"/> button and select the **Apply Dimension** option.

<img src="../img/widgets/apply-dimension-option.jpg" class="ms-docimage"  style="max-width:450px;"/>

* Connect the filter to the layer by clicking the <img src="../img/button/connect-button.jpg" class="ms-docbutton"/> button.

<img src="../img/widgets/connect-dimension.jpg" class="ms-docimage"  style="max-width:450px;"/>

> * To connect the filter to the map timeline, first click the <img src="../img/button/settings2.jpg" class="ms-docbutton"/> button, then enable the **2 way synchronization**.

Reviewer: ```suggestion
* To connect the filter to the map timeline, first click the <img src="../img/button/settings2.jpg" class="ms-docbutton"/> button, then enable the **2-way synchronization**.
```
